src/tablemodel.cpp

```diff
@@ -455,3 +455,41 @@ QColor TableModel::getMsgBackgroundColor(const std::optional<QDltMsg>& msg, int
 
     return brushColor; // this is the default background color
 }
+
+QString TableModel::getToolTipForFields(FieldNames::Fields cn)
+{
+    switch(cn){
+    case(FieldNames::Time): return "Detailed representation of time and date when the log entry was recorded";
+    case(FieldNames::TimeStamp): return "Displays the time when the log message was generated";
```

Comment:
I would probably also mention in what units and relative to what epoch it is given.